zellij-server/src/screen.rs

```diff
@@ -370,7 +370,9 @@ impl Screen {
             tab_data.push(TabInfo {
                 position: tab.position,
                 name: tab.name.clone(),
+                panes: tab.get_pane_num(),
```

Comment:
When the full screen is active, all panes that are not full-screen are placed in tab.panes_to_hide... I think we can count them instead of doing this. What do you think?

Reply:
I find it more convenient to use panes_to_hide, so I'll use that.